onnxruntime/core/providers/cuda/object_detection/non_max_suppression_impl.cu

```diff
@@ -14,16 +14,16 @@ limitations under the License.
 ==============================================================================*/
 /* Modifications Copyright (c) Microsoft. */
 
+#include <thrust/device_vector.h>
+#include <thrust/execution_policy.h>
+
 #include "non_max_suppression_impl.h"
```

Comment:
the related header should be first
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes